_locales/cs/pxt-TPBot-strings.json

```diff
@@ -0,0 +1,44 @@
+{
+  "TPBot.DriveDirection.Backward|block": "vzad",
+  "TPBot.DriveDirection.Forward|block": "vpřed",
+  "TPBot.DriveDirection.Left|block": "vlevo",
+  "TPBot.DriveDirection.Right|block": "vpravo",
+  "TPBot.LineSide.Left|block": "levý",
+  "TPBot.LineSide.Right|block": "pravý",
+  "TPBot.LineState.Black|block": "černou",
+  "TPBot.LineState.White|block": "bílou",
+  "TPBot.MbEvents.Black|block": "černou",
+  "TPBot.MbEvents.White|block": "bílou",
+  "TPBot.MbPins.Left|block": "levý",
+  "TPBot.MbPins.Right|block": "pravý",
+  "TPBot.ServoList.S1|block": "S1",
+  "TPBot.ServoList.S2|block": "S2",
+  "TPBot.ServoList.S3|block": "S3",
+  "TPBot.ServoList.S4|block": "S4",
+  "TPBot.SonarUnit.Centimeters|block": "cm",
+  "TPBot.SonarUnit.Inches|block": "palcích",
+  "TPBot.Sonarjudge.Greater|block": ">",
+  "TPBot.Sonarjudge.Less|block": "<",
+  "TPBot.TrackingState.L_R_line|block": "● ●",
+  "TPBot.TrackingState.L_R_unline|block": "◌ ◌",
+  "TPBot.TrackingState.L_line_R_unline|block": "● ◌",
+  "TPBot.TrackingState.L_unline_R_line|block": "◌ ●",
+  "TPBot.headlightClose|block": "vypni světlomety",
+  "TPBot.headlightColor|block": "nastav barvu světlometů na $color",
+  "TPBot.headlightRGB|block": "nastav barvu světlometů složenou z červené:%r zelené:%g modré:%b",
```

Comment:
This block is way too long, but I couldn't come up with a better way to translate it either.